zh/basic_concepts/scene.md

```diff
@@ -1,75 +1,46 @@
-<div class="langs">
-  <a href="#" class="btn" onclick="toggleLanguage()">中文</a>
-</div>
+# 场景(Scene)
 
-## Scene
-In your game you probably want a main menu, a few levels and an ending scene.
-How do you organize all of these into the separate pieces they are? You guessed
-it, `Scene`. When you think about your favorite movie you can see that it's
-distinctly broken down into scenes, or separate parts of the story line. If we
-apply this same thought process to games, we should come up with at least a few
-scenes no matter how simple the game is.
+在游戏开发过程中，你可能需要一个主菜单，几个关卡和一个结束场景。如何组织所有这些分开的部分？使用 __场景(Scene)__ ! 当你想到喜欢的电影时，你能观察到它是被分解为不同场景或不同故事线。现在我们对游戏开发应用这个相同的思维过程，你应该很容易就能想出几个场景。
 
-Taking another look at the familiar image from earlier:
+来看一张熟悉的图片：
 
-![](basic_concepts-img/2n_main.png "")
+![](../../en/basic_concepts/basic_concepts-img/2n_main.png "")
 
-This is a main menu and it is a single `Scene`. This scene is made up of
-several pieces that all fit together to give us the end result. Scenes are drawn
-by the __renderer__. The __renderer__ is responsible for rendering sprites and
-other objects into the screen. To better understand this we need to talk a bit
-about the __scene graph__.
+这是一个主菜单场景, 这个场景是由很多小的对象拼接而成, 所有的对象组合在一起, 形成了最终的结果. 场景是被 __渲染器(renderer)__ 画出来的. 渲染器负责渲染精灵和其它的对象进入屏幕. 为了更好的理解这个过程, 我们需要讨论一下 __场景图__.
 
-### Scene Graph
-A __scene graph__ is a data structure that arranges a graphical scene. A
-__scene graph__ contains `Node` objects in a tree (yes, it is called
-__scene graph__, but it is actually represented by a __tree__) structure.
+## 场景图(Scene Graph)
```

Comment:
beg your review, I don't know Scene Graph clearly @minggo

Reply:
What is your question?